test/Sentry.Tests/Protocol/UserFeedbackTests.cs

```diff
@@ -1,36 +0,0 @@
-namespace Sentry.Tests.Protocol;
-
-public class UserFeedbackTests
```

Comment:
question: testing `SentryFeedback`

I believe we are lacking explicit/dedicated `SentryFeedbackTests.cs`.
Are they needed? Or do we cover the `SentryFeedback` serializable already through other tests?